src/cache_invalidate.c

```diff
@@ -58,19 +58,6 @@ cache_invalidate_relcache_all(void)
 	ts_bgw_job_cache_invalidate_callback();
 }
 
-static inline void
-cache_invalidate_syscache_all(void)
-{
-	ts_cm_functions->cache_syscache_invalidate(PointerGetDatum(NULL), 0, 0);
```

Comment:
Do we need  ts_cm_functions-> cache_syscache_invalidate anymore?

Reply:
Yes, it is still used by the `cache_invalidate_syscache_callback`